tsl/test/expected/data_node.out

```diff
@@ -1641,6 +1641,26 @@ SELECT sum(device) FROM dist_test;
  846
 (1 row)
 
+SELECT * FROM timescaledb_experimental.chunk_replication_status;
+ hypertable_schema | hypertable_name |     chunk_schema      |       chunk_name       | desired_num_replicas | num_replicas |       replica_nodes       |     non_replica_nodes     
+-------------------+-----------------+-----------------------+------------------------+----------------------+--------------+---------------------------+---------------------------
+ public            | dist_test       | _timescaledb_internal | _dist_hyper_9_12_chunk |                    1 |            2 | {data_node_1,data_node_3} | {data_node_2}
+ public            | dist_test       | _timescaledb_internal | _dist_hyper_9_13_chunk |                    1 |            1 | {data_node_2}             | {data_node_1,data_node_3}
+ public            | dist_test       | _timescaledb_internal | _dist_hyper_9_14_chunk |                    1 |            1 | {data_node_3}             | {data_node_1,data_node_2}
+ public            | dist_test       | _timescaledb_internal | _dist_hyper_9_15_chunk |                    1 |            1 | {data_node_1}             | {data_node_2,data_node_3}
+(4 rows)
+
+-- Check that they can be called from inside a procedure without
+-- generating warnings or error messages (#3495).
+CREATE OR REPLACE PROCEDURE wrapper(regclass, text, text)
+AS $$
+BEGIN
+    CALL timescaledb_experimental.copy_chunk($1, $2, $3);
```

Comment:
For completeness, we should also add a wrapper call to `timescaledb_experimental.move_chunk`

Reply:
They are using the same basic function, but yeah, it makes sense to test both just to make sure that any changes does not cause a regression.